jmix-gridexport/gridexport-flowui/src/main/java/io/jmix/gridexportflowui/exporter/AbstractAllRecordsExporter.java

```diff
@@ -90,9 +136,108 @@ public LoadContext generateLoadContext(DataUnit dataUnit) {
         PropertyCondition lastPkCondition = PropertyCondition.createWithParameterName(primaryKeyName,
                 PropertyCondition.Operation.GREATER, LAST_LOADED_PK_CONDITION_PARAMETER_NAME);
         wrappingCondition.add(lastPkCondition);
-        loadContext.getQuery().setCondition(wrappingCondition);
+        query.setCondition(wrappingCondition);
+        query.setFirstResult(0);
 
         return loadContext;
     }
+
+    /**
+     * Method loads all entity instances associated with the given {@code dataUnit} and calls the
+     * {@code entityExporter} to export each loaded entity instance. Creation of the output file object is the
+     * responsibility of the function. Data is loaded in batches, the batch size is configured by the
+     * {@link GridExportProperties#getExportAllBatchSize()}.
+     *
+     * @param dataUnit       data unit linked with the data
+     * @param entityExporter function that is applied to each loaded instance
+     */
+    public void exportAll(DataUnit dataUnit, EntityExporter entityExporter) {
+        Preconditions.checkNotNullArgument(entityExporter, "Cannot export all rows. EntityExporter can't be null");
```

Comment:
if you check one parameter, then check another, because they both must be not null. 

Additionally, add `package-info.java` to `exporter`, `exporter.excel` and `exporter.json` packages. For some reason it's missed.